### DISCUSSION TITLE
Bedrock and Agents

### DISCUSSION THREAD

**Discussion by ala85** [danny-avila/LibreChat#4943] [Help Wanted]
I am aware there is no support for Bedrock Agents, but can anyone share then how to use the LibreChat agent framework to then use Bedrock models? Is it feasible with current code?

**Comment by danny-avila** [comment]
It's simple, just follow this: https://www.librechat.ai/docs/features/agents

agents need to be enabled if using `ENDPOINTS` environment variable, i.e. `ENDPOINTS=agents,#...rest`

![chrome_LcNUTX1fNM](https://github.com/user-attachments/assets/a4177186-ee57-4f7d-a8cc-041231ea3bc1)
